pkg/kn/commands/route/route.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/knative/client/pkg/kn/commands"
+	"github.com/spf13/cobra"
+)
+
+func NewRouteCommand(p *commands.KnParams) *cobra.Command {
+	routeCmd := &cobra.Command{
+		Use:   "route",
+		Short: "Route command group",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
@maximilien i was hoping to avoid copy-pasta (and associated tests) every time we add a command.

Reply:
Yes, dude. On this today. @navidshaikh you can remove this as #111 will fix for all command group that do not include a `RunE` and have subcommands.

Reply:
removed, thanks @maximilien!